src/modules/featureFlag/featureFlag.ts

```diff
@@ -0,0 +1,54 @@
+import Configstore from 'configstore'
+import { join } from 'path'
+import { CLIPreTasks } from '../../CLIPreTasks/CLIPreTasks'
+
+export interface IFeatureFlag {
+  storeFilePath: string
+  getLastFeatureFlagUpdate: () => number
+  setLastFeatureFlagUpdate: (date: number) => void
+  getAllFeatureFlagInfo: () => Record<string, any>
+  getFeatureFlagInfo: <T>(flagName: string) => T
+  setFeatureFlagInfo: (info: Record<string, any>) => void
+}
```

Comment:
Sorry, but I didn't understand why you created that interface

Reply:
Removed, actually don't need this